consensus-types/blocks/setters.go

```diff
@@ -125,7 +125,7 @@ func (b *SignedBeaconBlock) SetBlobKzgCommitments(c [][]byte) error {
 	switch b.version {
 	case version.Phase0, version.Altair, version.Bellatrix, version.Capella:
```

Comment:
I read these better with an if-else (which is also better performing)
```
if b.version < version.Deneb {
} else if b.version <= version.Electra {
}
else {
    bail out
}
```

Reply:
```go
// SetBlobKzgCommitments sets the blob kzg commitments in the block.
func (b *SignedBeaconBlock) SetBlobKzgCommitments(c [][]byte) error {
	if b.version < version.Deneb {
		return consensus_types.ErrNotSupported("SetBlobKzgCommitments", b.version)
	}
	b.block.body.blobKzgCommitments = c
	return nil
}
```

It can be so simple